doc/flatpak-metadata.xml

```diff
@@ -133,31 +147,191 @@
+                                Flatpak.
+                                In particular, this shares
+                                <filename>/home</filename>,
+                                <filename>/media</filename>,
+                                <filename>/opt</filename>,
+                                <filename>/run/media</filename> and
+                                <filename>/srv</filename> if they exist.
+                            </para>
+                            <para>
+                                <filename>/dev</filename> is not shared:
+                                use <option>devices=all;</option> instead.
+                            </para>
+                            <para>
+                                Parts of <filename>/sys</filename> are always
+                                shared. This option does not make additional
+                                files in /sys available.
+                            </para>
+                            <para>
+                                These other reserved directories are
+                                currently excluded:
+                                <filename>/app</filename>,
+                                <filename>/bin</filename>,
+                                <filename>/boot</filename>,
+                                <filename>/etc</filename>,
+                                <filename>/lib</filename>,
+                                <filename>/lib32</filename>,
+                                <filename>/lib64</filename>,
+                                <filename>/proc</filename>,
+                                <filename>/root</filename>,
+                                <filename>/run</filename>,
+                                <filename>/sbin</filename>,
+                                <filename>/tmp</filename>,
+                                <filename>/usr</filename>,
+                                <filename>/var</filename>.
+                            </para>
+                            <para>
+                                Available since 0.3.
+                            </para></listitem></varlistentry>
+
+                            <varlistentry><term><option>xdg-desktop</option>,
+                                <option>xdg-documents</option>,
+                                <option>xdg-download</option>,
+                                <option>xdg-music</option>,
+                                <option>xdg-pictures</option>,
+                                <option>xdg-public-share</option>,
+                                <option>xdg-videos</option>,
+                                <option>xdg-templates</option>
+                            </term><listitem><para>
+                                <ulink url="https://www.freedesktop.org/wiki/Software/xdg-user-dirs/"
+                                  >freedesktop.org special directories</ulink>.
```

Comment:
Nitpick: Funky line wrapping going on here and in a few other places.

Reply:
Some of that is deliberate - if I did the break like

```
<ulink url="this is too long">
  link text
</ulink>
```

then the link text "officially" starts and ends with a space, whereas breaking inside the tag avoids that.